.fixtures.yml

```diff
@@ -1,8 +1,8 @@
 fixtures:
-  forge_modules:
-    apt: 'puppetlabs-apt'
-    augeas: 'camptocamp-augeas'
-    augeasproviders_core: 'herculesteam-augeasproviders_core'
-    stdlib: 'puppetlabs-stdlib'
+  repositories:
```

Comment:
because we do that in every other module too. We always test against master, not the last release.